PublicMapping/districtbuilder
# Add support for multi member districts

## Overview

Adds support for making a project use multi-member districts when creating / importing it.

Multi-member projects will have different population targets, based on the number of representatives per district (e.g. we're targeting equal population per _representative_, which thus far has been the same as equal population per district).

### Checklist


### Demo

Creating a multi-member project:
![create-multi-member](https://user-images.githubusercontent.com/4432106/140571083-90c5da4c-31c9-451a-9f17-1ca1651a2bf4.gif)

Importing:
![import-multi-member](https://user-images.githubusercontent.com/4432106/140571102-2116c334-4411-4c3d-a302-cab32ecb18c0.gif)

Evaluate mode Equal pop. detail:
![image](https://user-images.githubusercontent.com/4432106/140572731-ddd06747-2a4d-4e42-ad38-5133c21c5b71.png)


### Notes

The original issue didn't call for adding `multi-member` support to Project Templates or Chambers, but it was a relatively small part of this overall feature, and something we've discussed wanting to support.

Similarly, we hadn't fully considered how multi-member would affect Evaluate mode (which calculates equal pop. / district) or the Tour (which references the district target pop.).

For the tour, I simply switched to referencing target pop. / _representative_, which will be the same as a 1-member district whether or not multi-member districts are in use (the only time this might get confusing is when creating a map with no single-member districts).

For Evaluate mode, I switched to showing target pop. / representative as above, as well as added a "Number of reps" column.

## Testing Instructions

- `scripts/update`
- Try creating / importing a project without setting the multi-member checkbox - there shouldn't be any regressions
- Create a project with multi-member districts:
   - Calculations on the sidebar & evaluate equal population should take your multi-member districts into account
- Creating a project w/ a Chamber w/ multi-member districts, or for a a project template w/ multi-member districts should copy their `numberOfMembers` field over to your project

Closes #1024
